vendor/engines/secure_rooms/lib/secure_rooms/engine.rb

```diff
@@ -11,11 +11,15 @@ class Engine < ::Rails::Engine
       ViewHook.add_hook "users.show",
                         "additional_user_fields",
                         "secure_rooms/shared/card_number_form_field"
+
+      ViewHook.add_hook "admin.shared.tabnav_users",
+                        "after",
+                        "secure_rooms/shared/tabnav_users"
     end
 
-    initializer :append_migrations do |app|
```

Comment:
The `:append_migrations` initializer is duplicated (expand below to see), so this removes one of them.